internal/cmdrpkgcopy/command.go

```diff
@@ -63,10 +63,10 @@ func newRunner(ctx context.Context, rcg *genericclioptions.ConfigFlags) *runner
 		cfg: rcg,
 	}
 	c := &cobra.Command{
-		Use:     "edit SOURCE_PACKAGE NAME",
+		Use:     "copy SOURCE_PACKAGE NAME",
```

Comment:
we can consider keeping edit as an alias (though we can start with copy and seeing where it gets us)

Reply:
Yeah, adding edit as an alias is a good idea.